app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -44,20 +46,21 @@ class FxaRepo(
 ) {
 
     enum class AccountState {
-        // TODO: Are these states accurate?
         // TODO: Later, may need "failed to login": https://github.com/mozilla-mobile/android-components/issues/3712
-        AUTHENTICATED,
-        AUTHENTICATED_NO_PROFILE, // todo: necessary? profile seems to be fetched async so yes?
+        AUTHENTICATED_WITH_PROFILE, // After the profile is fetched async
+        AUTHENTICATED_NO_PROFILE, // Before the profile is fetched async
         NEEDS_REAUTHENTICATION,
-        NOT_AUTHENTICATED
-        // TODO: do we need initial state before state is fetched from disk?
+        NOT_AUTHENTICATED // Initial state
     }
 
-    // TODO: set state accurately, choose correct subject, set initial state, etc.
-    val accountStateDontUseMeYet = BehaviorSubject.create<AccountState>()
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
```

Comment:
nit: typically I important each VisibleForTesting value so this can be: `@VisibleForTesting(otherwise = NONE)`